File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size(),l=0,r=1,c=1;
        if(n==1) return 1;
        while(r<n){
            while(r<n && nums[l]==nums[r]){
                r++;
            }
            if(r<n){
                l++;
                nums[l]=nums[r]; 
                r++;
                c++;
            }
        }
        return c;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int count = 1,l=0,r=0;
        for(r=0;r<nums.length;r++){
            if(nums[r]>nums[l]){
                l++;
                nums[l]=nums[r];
                
            }            
        }
        return l+1;
    }
}